proposals/lip-0037.md

```diff
@@ -60,7 +60,7 @@ Using 4 bytes instead of 32 bytes as in [LIP 0009][lip-0009] has the advantage t
 
 | Name | Type | Validation | Description |
 |------|------|------------|-------------|
-| `ChainID` | `bytes` | Must be of length `CHAIN_ID_LENGTH`. | ID of a chain. |
+| `ChainID` | `bytes` | Must be of length `CHAIN_ID_LENGTH`. | ID of a chain which is `chainID.to_bytes(4, 'big')` i.e. big endian uint32 serialization of `chainID`.|
```

Comment:
What do you mean here? The chainID is always of type bytes, why do you need to serialize it from uint?

Reply:
I was trying to explain that we have now converted it into the bytes. Earlier it was an integer value. But now I have removed it, seems confusing.